### DISCUSSION TITLE
Pebble 's base design.

### DISCUSSION THREAD

**Discussion by PebbleCat-Studios** [photopea/photopea#6372] [Show and tell]
I'm posting this here because I saw there was a tag that no one used, so FIRST!

<Made entirely in Photopea!>

![Pebble V3](https://github.com/photopea/photopea/assets/146106615/b9a74d51-ab3c-4618-8ed8-f248e198ba79)

**Comment by PebbleCat-Studios** [comment]
Any input is highly appreciated! (Even constructive criticism. :3)

**Reply by PebbleCat-Studios** [reply]
I know the outline is rough.

**Reply by camoody1** [reply]
You spelled "design" wrong. Otherwise, I really like the giraffe you've drawn. 👍🏼😊

**Reply by PebbleCat-Studios** [reply]
Thanks, I fixed the mistake.

**Comment by photopea** [comment]
That looks great! If you record a screen of you painting it a Photopea, I can share that video at our social networks :)

**Reply by PebbleCat-Studios** [reply]
I'll see what I can do. : )
